src/components/Popover.js

```diff
@@ -1,110 +1,84 @@
-import React from 'react';
+import React, { useState, useRef } from 'react';
 import PropTypes from 'prop-types';
 import MuiPopover from '@material-ui/core/Popover';
 import IconButton from '@material-ui/core/IconButton';
-import { findDOMNode } from 'react-dom';
 import CloseIcon from '@material-ui/icons/Close';
 
-class Popover extends React.Component {
-  state = {
-    open: false,
-  };
+const Popover = ({ className, trigger, refExit, content, ...providedProps }) => {
+  const [isOpen, open] = useState(false);
+  const anchorEl = useRef(null);
 
-  UNSAFE_componentWillMount() {
-    this.anchorEl = null;
-  }
+  const handleClick = event => {
+    anchorEl.current = event.currentTarget;
```

Comment:
ref : https://material-ui.com/api/popover/
anchorEl | HTML element\| func |   | A HTML element, or a function that returns it. It's used to set the position of the popover.
-- | -- | -- | --


The setting time(**Assign Html to anchorEl [useRef]**) has been changed by using A.